data/usegalaxy.eu.yaml

```diff
@@ -0,0 +1,39 @@
+title: European Galaxy Server
+provider: European Galaxy Project, NFDI4Plants, NFDI4BioImaging, EOSC
+service_url: https://live.usegalaxy.eu
+support: contact@usegalaxy.eu  
+health_api_url: https://status.galaxyproject.org
+documentation_url: https://training.galaxyproject.org
+target_group_open_for: Open for everyone
+login_process: Login via OpenIDC (LS-AAI, DataPLANT)
+features:
+  version: Galaxy 23.0
+  kernels: ["Python", "Julia", "R", "Ansible"]
+  extensions: []
+  proxy_apps: [Jupyter, Jupyter ML, RStudio, MGnify, AskOMICS, 
+    BAM-IOBICF-IOBIO, CellXGene, Ethercalc, Phinch, MetaShARK, Wallace,
+    iSEE, Wilson, Jupyter for Material Science, Pangeo Jupyter, Climate Jupyter, Panoply netCDF Viewer, HiGlass
+    Pavian, OpenRefine, Neo4J, GeoExplorer, NEAL]
```

Comment:
We don't have a list with those descriptions. We certainly have - its just hard to keep those updated. Those images are largely community maintained and we (the provider) should not really know details - as long as we trust the community.

I removed the Jupyter* deployments and added a few more we added over the last month.

Reply:
`Those images are largely community maintained and we (the provider) should not really know details - as long as we trust the community.`
Interesting! :-) Just, because I'm curious: is there a process/ place how the community creates/ maintains the images and when/ how they are handed to you? For example, could I also give you an image with my favorite research software and you would deploy it on Galaxy EU? :-)

Reply:
Yes, that is how it works. Give me a Docker image and a metadata description of the image (container, name, port, inputs?, output? small description, help-text, citations etc ...) and we will deploy it.

I have written the procedure down, it was long overdue: https://usegalaxy-eu.github.io/operations/interactive_tools.html

Do you have any image that you would like to see ;)